spec/controllers/api/v0x0/endpoints_controller_spec.rb

```diff
@@ -10,12 +10,12 @@
 
   it "post /endpoints creates an Endpoint" do
     headers = { "CONTENT_TYPE" => "application/json" }
-    post(api_v0x0_endpoints_url, :params => {:host => "example.com", :source_id => source.id.to_s, :tenant_id => tenant.id.to_s}.to_json)
```

Comment:
These actually need to be strings because JSON can't handle bigints I think

Reply:
That's correct...we intentionally have ids as strings in the outgoing payload.